client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -28,14 +29,31 @@ function EditorDrawerTaxonomies( { siteId, postType, taxonomies } ) {
 					return;
 				}
 
-				return (
-					<Accordion
-						key={ taxonomy.name }
-						title={ taxonomy.label }
-						icon={ <Gridicon icon="tag" /> }>
-						Work in Progress
-					</Accordion>
-				);
+				if ( taxonomy.hierarchical ) {
+					return (
+						<Accordion
+							key={ taxonomy.name }
+							title={ taxonomy.label }
+							icon={ <Gridicon icon="category" /> }
+						>
+							Work in Progress
+						</Accordion>
+					);
+				} else {
+					return (
+						<Accordion
+							key={ taxonomy.name }
+							title={ taxonomy.label }
+							icon={ <Gridicon icon="tag" /> }
+						>
+							<TermTokenField
+								postTerms={ postTerms }
```

Comment:
Since `TermTokenField` is also connected, any reason to just not grab `postTerms` there and not pass it in here as a prop?